site/server/test/ArbitraryInstances.scala

```diff
@@ -33,7 +33,7 @@ trait ArbitraryInstances extends Assertions {
       email ← Gen.alphaStr
     } yield Request(
       login = login.toString,
-      name = name,
+      name = Some(name),
```

Comment:
Please prefer the option type instead of some because it's different behavior on null values